ts/IterableInbox.tsx

```diff
@@ -1,25 +1,58 @@
-'use strict'
-
+'use strict';
 import React, { Component, useState } from 'react'
 import { Text, View, StyleSheet } from 'react-native'
 import IterableInboxMessageList from './IterableInboxMessageList'
 import IterableInboxEmptyState from './IterableInboxEmptyState'
+import IterableInboxMessageDisplay from './IterableInboxMessageDisplay'
 import sampleMessages from './sampleMessageData.js'
 
 const IterableInbox = () => {
-   const message = "Inbox";
-   const [messages, setMessages] = useState(sampleMessages);
+   const inboxTitle = "Inbox"
+   const [isDisplayMessage, setIsDisplayMessage] = useState(false)
+   const [selectedMessageIdx, setSelectedMessageIdx] = useState(0)
+   const [messages, setMessages] = useState(sampleMessages)
+
+   let selectedMessage = messages[selectedMessageIdx]
+
+   function handleMessageSelect(index: number, messages: Array<any>) {
+      if (!messages[index].read) {
+         messages[index].read = true
+      }
+      setMessages([...messages])
+      setIsDisplayMessage(true)
+      setSelectedMessageIdx(index)
+   }
```

Comment:
so this is what i was talking about with the duplicate code from [the other pr](https://github.com/Iterable/react-native-sdk/pull/175/files), where you've actually made changes to how this will work. 

having this exact same code duplicated in two places means that I am no longer sure what I am reviewing. I guess, merge the other PR first so that those changes over there don't get overwritten.